src/build/sorter.ts

```diff
@@ -0,0 +1,67 @@
+import { readFileSync } from "fs";
+import { parse, ParseError, visit } from "jsonc-parser";
+import { URL } from "url";
+
+// Recursively sort object keys
+function sortObjectKeys<T>(obj: T): T {
+  if (obj && typeof obj === "object" && !Array.isArray(obj)) {
+    const sorted: Record<string, unknown> = {};
+    for (const key of Object.keys(obj).sort()) {
+      sorted[key] = sortObjectKeys((obj as any)[key]);
+    }
+    return sorted as T;
+  }
+  return obj;
```

Comment:
This do not need to return an object, maybe do not return anything and throw if it's not sorted.